scripts/byron-to-alonzo/mkfiles.sh

```diff
@@ -319,12 +320,41 @@ echo
 ls -1 shelley/*
 echo "====================================================================="
 
-echo "Generated genesis.json:"
+echo "Generated shelley/genesis.json:"
 echo
 cat shelley/genesis.json
 echo
 echo "====================================================================="
 
+# TODO alonzo: Should this be generated by cardano-cli genesis create?
```

Comment:
So I think we should modify `cardano-cli genesis create` to also create an alonzo genesis file. It shouldn't be too much of a change as most of the right bits are already there.

Reply:
Done!